Cargo.toml

```
@@ -195,7 +198,7 @@ borsh = { version = "1.5.0", features = ["derive"] }
 bytes = "1.6.0"
 cfg-if = "1.0.0"
 chrono = "0.4.38"
-clap = "4"
+clap = { version="4.5.39", default-features = false, features = ["derive"] }
```

Don't know how `taplo` did not get this

```suggestion
clap = { version = "4.5.39", default-features = false, features = ["derive"] }
```